.github/workflows/unix_unit_tests.yml

```diff
@@ -58,11 +58,15 @@
           shell: bash -el {0}
           run: |
             flake8 --count hnn_core
-        - name: Test with pytest
+        - name: Test embarrassingly parallel tests with pytest
+          shell: bash -el {0}
+          run: |
+            python -m pytest ./hnn_core/tests/ -m "not already_parallel" -n auto --cov=hnn_core --cov-report=xml
+        - name: Test true parallel tests with pytest
           shell: bash -el {0}
           run: |
-            python -m pytest ./hnn_core/tests/  --cov=hnn_core --cov-report=xml
+            python -m pytest ./hnn_core/tests/ -m "already_parallel" --cov=hnn_core --cov-report=xml --cov-append
```

Comment:
maybe we should add this to the Makefile: https://github.com/jonescompneurolab/hnn-core/blob/master/Makefile

then you use the same command locally too. E.g.:

```sh
$ make already_parallel_tests
```

Reply:
and the `make test` has a dependency on `already_parallel_tests` and `parallel_tests` like so:

```sh
tests : already_parallel parallel_tests
```
could you come up with slightly better names so the distinction between the two categories are clear?

Reply:
What about `@pytest.mark.uses_mpi`? That way it is unique, easy to understand (and `grep`), but also distinct from the other valid decorator `@requires_mpi4py`.

Reply:
yes, I like that!